src/jit/flowgraph.cpp

```diff
@@ -17776,6 +17776,10 @@ BasicBlock* Compiler::fgAddCodeRef(BasicBlock* srcBlk, unsigned refData, Special
 #if defined(UNIX_X86_ABI)
         codeGen->setFrameRequired(true);
         codeGen->setFramePointerRequiredGCInfo(true);
+        add->acdStkLvl = 0;
+#ifdef DEBUG
+        add->acdDstBlk->bbTgtStkDepth = 0;
+#endif
 #else  // !defined(UNIX_X86_ABI)
```

Comment:
``fgThrowHlpBlkStkLevel`` called from ``CodeGen::genAdjustStackLevel`` uses this, but it seems that this is not relevant  for EBP frame cases. I'll check and update PR if it is really irrelevant.